cmd/webhook_v1.go

```diff
@@ -71,12 +71,18 @@ This command is only available for v1 scaffolding project.
 				&model.Universe{},
 				input.Options{},
 				&manager.Webhook{},
-				&webhook.AdmissionHandler{Resource: o.res, Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
-				&webhook.AdmissionWebhookBuilder{Resource: o.res, Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
-				&webhook.AdmissionWebhooks{Resource: o.res, Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
-				&webhook.AddAdmissionWebhookBuilderHandler{Resource: o.res, Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
-				&webhook.Server{Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
-				&webhook.AddServer{Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
+				&webhook.AdmissionHandler{Resource: o.res,
+					Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
+				&webhook.AdmissionWebhookBuilder{Resource: o.res,
+					Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
+				&webhook.AdmissionWebhooks{Resource: o.res,
+					Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
+				&webhook.AddAdmissionWebhookBuilderHandler{Resource: o.res,
+					Config: webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}},
+				&webhook.Server{Config: webhook.Config{Server: o.server,
+					Type: o.webhookType, Operations: o.operations}},
+				&webhook.AddServer{Config: webhook.Config{Server: o.server,
+					Type: o.webhookType, Operations: o.operations}},
```

Comment:
Could we maybe create the config server beforehand and just pass it to the files?

```
fmt.Println("Writing scaffold for you to edit...")

webhookConfig := webhook.Config{Server: o.server, Type: o.webhookType, Operations: o.operations}

err = (&scaffold.Scaffold{}).Execute(
	&model.Universe{},
	input.Options{},
	&manager.Webhook{},
	&webhook.AdmissionHandler{Resource: o.res, Config: webhookConfig},
	&webhook.AdmissionWebhookBuilder{Resource: o.res, Config: webhookConfig},
	&webhook.AdmissionWebhooks{Resource: o.res, Config: webhookConfig},
	&webhook.AddAdmissionWebhookBuilderHandler{Resource: o.res, Config: webhookConfig},
	&webhook.Server{Config: webhookConfig},
	&webhook.AddServer{Config: webhookConfig},
)
```

It should be clearer, less verbose and easy to read